core/templates/pages/exploration-editor-page/modal-templates/help-modal.component.html

```diff
@@ -6,31 +6,31 @@ <h1>Need Help?</h1>
     Check out the resources in our help center or
     take a tour of the exploration editor.
   </p>
-  <a href="https://oppia.github.io/#/" target="_blank" rel="noopener" class="btn oppia-help-modal-button oppia-help-dismiss" ng-click="goToHelpCenter()">Visit the Help Center</a>
-  <button class="btn oppia-help-modal-button oppia-autofocus" ng-click="beginEditorTutorial()" autofocus>Take the Tour</button>
-  <button class="btn oppia-help-modal-button" ng-click="beginTranslationTutorial()">Translation Tour</button>
+  <a href="https://oppia.github.io/#/" target="_blank" rel="noopener" class="btn oppia-help-modal-button oppia-help-dismiss" (click)="goToHelpCenter()">Visit the Help Center</a>
+  <button class="btn oppia-help-modal-button oppia-autofocus" (click)="beginEditorTutorial()" autofocus>Take the Tour</button>
+  <button class="btn oppia-help-modal-button" (click)="beginTranslationTutorial()">Translation Tour</button>
 </div>
 <style>
   .oppia-help-modal-header-container {
     text-align: center;
   }
 
-  .oppia-help-modal .modal-body {
+  .modal-body {
```

Comment:
Why remove the `.oppia-help-modal`s?

Reply:
Ah got it. That's because the `<style>...</style>` would need to be moved to `oppia.css`, I believe. @srijanreddy98 @ashutoshc8101 -- what do you think is best in this case where `oppia-help-modal` is set as the `windowClass`? Should we move the styling to `oppia.css` or alter the HTML classes like done in this PR?

Reply:
Yes, the styles should be moved to `oppia.css`.

Reply:
https://user-images.githubusercontent.com/80846488/147189417-997d10e7-fd5e-4857-81c3-d1991580ae56.mp4

Reply:
@ashutoshc8101 -- is an additional `$rootScope.$applyAsync()` needed as part of this migration?

Reply:
Seems like a change detection issue. Using `$rootScope.$applyAsync()` here will help in fixing this issue.

Reply:
@JeeveshGarg have we figured this out yet? If you undo these changes and move them to `oppia.css`, does it render as expected?

Reply:
@EricZLou, by moving style to oppia.css we can't get desired result, also in below conversation with @srijanreddy98, he suggested to fit styles in html if possible.

i do the same.

Reply:
https://user-images.githubusercontent.com/80846488/149218586-06b9211c-adb7-4a88-ad38-627f721cf8ad.mp4

Reply:
updated video..